src/nvim/testdir/Makefile

```diff
@@ -10,22 +15,41 @@ export SHELL := sh
 export NVIM_PRG := $(NVIM_PRG)
 export TMPDIR
 
-SCRIPTS ?= \
-           test13.out             \
-           test14.out             \
-           test17.out             \
-           test24.out             \
-           test32.out             \
-           test37.out             \
-           test40.out             \
-           test42.out             \
-           test48.out             \
-           test49.out             \
-           test52.out             \
-           test53.out             \
-           test64.out             \
-           test73.out             \
-           test79.out             \
+ifeq ($(OS),Windows_NT)
+  SCRIPTS ?= \
+	test13.out             \
+	test14.out             \
+	test24.out             \
+	test37.out             \
+	test40.out             \
+	test42.out             \
+	test48.out             \
+	test49.out             \
+	test52.out             \
+	test53.out             \
+	test64.out             \
+	test73.out             \
+	test79.out             \
+
+else
+  SCRIPTS ?= \
```

Comment:
Since this is a superset of the Windows list, can we append to it? Something like:

```
ifneq ($(OS),Windows_NT)
    SCRIPTS := $(SCRIPTS) \
      ... \
      ...
```

Reply:
If the user wants to run one test only, doesn't that append the default tests?

Reply:
I missed `TESTNUM` which overrides `SCRIPT`. I think we can append to it.

Reply:
Oh, right. What about this:

```
SCRIPTS_DEFAULT = ... 
ifneq ($(OS),Windows_NT)
  SCRIPTS_DEFAULT := $(SCRIPTS_DEFAULT) \
    ...
endif

SCRIPTS ?= SCRIPTS_DEFAULT
``